### DISCUSSION TITLE
挂载百度网盘以后 无法用浏览器访问webdav服务

### DISCUSSION THREAD

**Discussion by whpu-yuancao** [AlistGo/alist#1035] [Q&A]
当用浏览器用过 webdav访问 挂载后的百度网盘，提示如下信息： 
Please connect using software that supports WebDAV instead of a browser.
Method Not Allowed

因此用clouddrive 也无法挂载该 webdav 服务。

**Comment by anwen-anyi** [comment]
https://alist-doc.nn.ci/docs/common-question/#webdav%E6%B5%8F%E8%A7%88%E5%99%A8%E6%89%93%E5%BC%80%E6%98%BE%E7%A4%BAinternal-server-error

**Comment by dreamaker** [comment]
同困扰,用filestation挂载webdav或者cloudrive挂载,上传文件都失败,创建文件夹没有问题

**Reply by whpu-yuancao** [reply]
你用clouddrive挂载webdav后，可以看到里面的目录，或者读文件嘛

**Comment by yiyanquan** [comment]
兄弟  你挂载百度网盘大文件怎么解决的   cloudflare禁止请求ip   本里代理带宽低  走自己流量    浏览器改请求头WEBDAV又怎么办   你解决了吗
